src/components/AddProblemToSection/AddProblemToSection.module.css

```diff
@@ -8,6 +8,8 @@
 }
 
 .AddProblemToSectionModalContent {
+    position: relative;
+    z-index: -1;
```

Comment:
Isn't it possible to fix without -1 z-index?

Reply:
I think not